@navikt/ds-react/src/help-text/HelpText.tsx

```diff
@@ -0,0 +1,98 @@
+import { Helptext as HelpTextIcon } from "@navikt/ds-icons";
+import { Placement } from "@popperjs/core";
+import cl from "classnames";
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+import mergeRefs from "react-merge-refs";
+import { Popover, useId } from "..";
+
+export interface HelpTextProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  /**
+   * Component content
+   */
+  children: React.ReactNode;
+  /**
+   * Tekst shown on mouseover and by screenreaders
+   * @default "hjelp"
+   */
+  title?: string;
+  /**
+   * Placement of popover
+   * @default "top"
+   */
+  placement?: Placement;
+}
+
+const HelpText = forwardRef<HTMLButtonElement, HelpTextProps>(
+  (
+    { className, children, placement = "top", title = "hjelp", ...rest },
+    ref
+  ) => {
+    const buttonRef = useRef<HTMLButtonElement | null>(null);
+    const mergedRef = mergeRefs([buttonRef, ref]);
+    const popoverRef = useRef<HTMLDivElement | null>(null);
+    const wrapperRef = useRef<HTMLDivElement | null>(null);
+
+    const [open, setOpen] = useState(false);
+    const popoverId = `popover-${useId()}`;
+
+    useEffect(() => {
+      open && popoverRef.current?.focus();
+    }, [open]);
+
+    useEffect(() => {
+      const handleKeydown = (e: KeyboardEvent) => {
+        e.key === "ESC" && setOpen(false);
```

Comment:
Jeg ser at `Popover.tsx` har kode for dette, og begge trenger vel ikke å implementere funksjonaliteten:
```js
useEventLister(
  "keydown",
  useCallback((e: KeyboardEvent) => e.key === "Escape" && close(), [close])
);
```

Reply:
👍